src/components/ProviderSettings/ProviderSettingsRenderer.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+import React from 'react';
+import styled from 'styled-components';
+import NetworkSelect from '../NetworkSelect/NetworkSelect';
```

Comment:
As we are following the module folder approach, it would be better exporting the `NetworkSelect` component on `index.js` of the folder so you could import it using `import NetworkSelect from '../NetworkSelect'`;

Reply:
Agreed. I was confused